addon/globalPlugins/updateChannel.py

```diff
@@ -88,19 +122,14 @@ def getAvailableUpdates(self, currentChannel):  # noqa C901
 				break
 			if channel == "default" or not channel:
 				continue
+			versionInfo.updateVersionType = channel
 			try:
-				versionInfo.updateVersionType = channel
-				# Workaround for issue 3
-				if originalChannel == "snapshot:alpha" and originalChannel != channel:
-					versionInfo.version = str(versionInfo.version_year) + "." + str(versionInfo.version_major) + "."\
-					+ str(versionInfo.version_minor)
 				self.availableUpdates[channel] = updateCheck.checkForUpdate()
+			except RuntimeError:  # Thrown by `updateCheck.checkForUpdate`
+				self.availableUpdates[channel] = -1  # An error occurred
+			else:
 				if not self.availableUpdates[channel]:
 					self.availableUpdates[channel] = 1  # Already updated
-			except Exception:
```

Comment:
This is not strictly related, however I wasted a lot of time trying to understand why my change did not worked, and it turned out I made a stupid typo, yet the exception got swallowed making debugging unnecessarily difficult. Please try  to avoid broad except statements like these.